public/js/src/enketo-webform.js

```diff
@@ -43,8 +43,9 @@ if ( settings.offline ) {
         .catch( _showErrorOrAuthenticate );
 } else {
     console.log( 'App in online-only mode.' );
-    initTranslator( survey )
-        .then( connection.getFormParts )
+    store.init()
```

Comment:
This probably speaks to the question about support in #137, it’s odd to initialize `store` here, but it was necessary to enable that support. I think it would be unusual not to have this feature parity, but if this isn’t preferred (or if there’s a better way to handle it) I’m all ears.

Reply:
Also, it occurs to me that some changes here may also be relevant to `enketo-webform-edit.js`? Unclear as I’m unsure how I might access that, I was hesitant to make changes there without tests or manual validation.